src/pages/amman/configuration.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Configuration
+metaTitle: Amman - Configuration
+description: Configuring Amman local validator toolkit.
+---
+
+When executed Amman will look for the configuration `.ammanrc.js` in the project root. If this file is not present Amman will load up with a default configuration.
+
+The config should be a JavaScript module exporting 'validator' with any of the below
+properties:
+
+- killRunningValidators: if true will kill any solana-test-validators currently running on the system.
```

Comment:
Nit: can you bold these Args so they stand out,m

Reply:
Added.